### PR TITLE
Project pull request

### PR DESCRIPTION
**Pull request by tufoinnkuo10**
This milestone was performed as follows;
- We were three of us in this project
- A kanban board was created and work was shared equally
- We worked using react and redux
- User data gotten from the API was not preserved
- A testing library was set up for all sections including the profile section
- We followed the git-flow and worked with the dev branch
- We used filter() to render the reserve list of the different sections
- All four test for the four sections passed and linter errors were checked as well

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,42 @@
...
+# Live links
+> [![Netlify Status](https://api.netlify.com/api/v1/badges/20d639fc-3987-45d5-bcf6-3a0db6782f98/deploy-status)](https://app.netlify.com/sites/space-ships-and-missions/deploys)
+
+> [LIVE PREVIEW](https://space-ships-and-missions.netlify.app/)
+
+
+# Authors
+- TUFOIN NKUO
+- AZARIA SAIDI(https://github.com/azy64)
+- LILLIAN FRANCIS
```
Comment (DeliceLydia) [inline README.md]: - [ ] I suggest that you add all authors(Github, LinkedIn etc) accounts to make your README professional

Comment (DeliceLydia) [reply README.md]: - [ ] Please remove all unnecessary scripts you can use this [link](https://github.com/microverseinc/readme-template)

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._




### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @tufoinnkuo10 @azy64 @lilyfrancis  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] Consider closing all open issues since your project is complete. All issues have been resolved.

Cheers and Happy coding!👏👏👏

**Pull request merged**